eng/Publishing.props

```diff
@@ -1,6 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <PropertyGroup>
-    <PublishingVersion>3</PublishingVersion>
-  </PropertyGroup>
+
+  <ItemGroup>
+    <ItemsToPushToBlobFeed Include="$(ArtifactsPackagesDir)**\*.nupkg"
+                           IsShipping="true"
+                           UploadPathSegment="Runtime"
```

Comment:
@NikolaMilosavljevic I don't think this line is necessary and should probably be deleted. Please also add a tracking issue for investigating why packages are not in the expected subfolders.

Reply:
Created an issue to fix this and remove custom publishing infra: https://github.com/dotnet/source-build/issues/4235